File: MyClub/UI/myclub_desktop/lib/models/position.dart
class Position {
  final int id;
  final String name;
  final bool isPlayer;

  Position({
    required this.id,
    required this.name,
    required this.isPlayer,
  });

  factory Position.fromJson(Map<String, dynamic> json) {
    return Position(
      id: json['id'] as int,
      name: json['name'] as String,
      isPlayer: json['isPlayer'] as bool? ?? true,
    );
  }

  Map<String, dynamic> toJson() {
    return {
      'id': id,
      'name': name,
      'isPlayer': isPlayer,
    };
  }
}

File: MyClub/MyClub.Services/Database/Entities/Position.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using Microsoft.EntityFrameworkCore.Internal;

namespace MyClub.Services.Database
{
    public class Position
    {
        [Key]
        public int Id { get; set; }

        [Required]
        [MaxLength(100)]
        public string Name { get; set; }

        public bool IsPlayer { get; set; } = true;

        public virtual ICollection<Player> Players { get; set; }
    }
} 